src/core_modules/capture-core/components/WidgetEnrollmentEventNew/WidgetEnrollmentEventNew.container.js

```diff
@@ -18,13 +42,69 @@ export const WidgetEnrollmentEventNew = ({ programId, stageId, onSave, ...passOn
 
     const formFoundation = stage.stageForm;
 
+    const onHandleSwitchTab = (newMode) => {
+        setWarningVisible(true);
```

Comment:
We should only show the warning if the user has interacted with the tab contents (e.g, changed any form values).